community-operators/federation/federation/0.0.5/federation.v0.0.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,299 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
```

Comment:
`metadata.annotations` are missing from this operator

Reply:
Which annotations are expected to be present?

Reply:
Here are some required fields for Operator Hub: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md
Here are some required fields for OLM under `spec`: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md#operator-metadata